### PR TITLE
feat: remove web3x from the code 🍻

### PR DESCRIPTION
**Pull request by nicosantangelo**
Depends on https://github.com/decentraland/decentraland-dapps/pull/257

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,186 +1,152 @@
...
-    "classnames": "^2.3.1",
-    "connected-react-router": "^6.9.1",
+    "connected-react-router": "^6.9.2",
     "content-hash": "^2.5.2",
-    "css-loader": "1.0.0",
-    "date-fns": "^2.14.0",
-    "dcl-catalyst-client": "^11.3.0",
-    "dcl-catalyst-commons": "^8.2.0",
-    "dcl-crypto": "^2.1.0",
+    "crypto-browserify": "^3.12.0",
```
[the remark this answers is not in the document]
Comment (nicosantangelo) [reply package.json:27]: Yes

File: package.json [hunk 2]
```diff
@@ -1,186 +1,152 @@
...
     "interface-datastore": "^0.7.0",
     "ipfs-unixfs-engine": "^0.35.4",
-    "ismobilejs": "^1.0.3",
+    "ismobilejs": "^1.1.1",
     "jszip": "^3.2.1",
-    "mini-css-extract-plugin": "0.4.3",
     "mousetrap": "^1.6.2",
-    "multihashing-async": "^0.8.0",
-    "optimize-css-assets-webpack-plugin": "5.0.1",
+    "os-browserify": "^0.3.0",
```
[the remark this answers is not in the document]
Comment (nicosantangelo) [reply package.json:44]: Yup

File: package.json [hunk 3]
```diff
@@ -1,186 +1,152 @@
...
-    "@types/mousetrap": "^1.6.0",
-    "@types/query-string": "^6.2.0",
+    "@types/interface-datastore": "^1.0.0",
+    "@types/mousetrap": "^1.6.3",
+    "@types/pull-stream": "^3.6.2",
     "@types/rc-slider": "^8.6.5",
     "@types/react": "^16.7.18",
-    "@types/react-dom": "^16.0.11",
+    "@types/react-copy-to-clipboard": "^4.3.0",
+    "@types/react-dom": "^18.0.5",
```
[the remark this answers is not in the document]
Comment (nicosantangelo) [reply package.json:84]: that is indeed inconvenient

File: src/lib/getModelData.ts [hunk 4]
```diff
@@ -144,13 +144,6 @@ export async function getModelData(url: string, options: Partial<Options> = {})
     scene.add(ambient)
 
     switch (thumbnailType) {
-      case ThumbnailType.FRONT: {
```
[the remark this answers is not in the document]
Comment (nicosantangelo) [reply src/lib/getModelData.ts:147]: Basically this was a bug 😅 the case below is actually from the front, this one from above 45degrees. It's only the light so nobody really realized

File: src/modules/item/sagas.ts [hunk 5]
```diff
@@ -600,7 +600,7 @@ export function* handleResetItemRequest(action: ResetItemRequestAction) {
     const newContents: Record<string, Blob> = yield Promise.all(
       Object.entries(entityContentsAsMap).map<Promise<[string, Blob]>>(async ([key, hash]) => [
         key,
-        await fetch(getCatalystContentUrl(hash)).then(res => res.blob())
+        await window.fetch(getCatalystContentUrl(hash)).then(res => res.blob())
```
[the remark this answers is not in the document]
Comment (nicosantangelo) [reply src/modules/item/sagas.ts:603]: Correct, tho I just tried again with CRA 4 (current) and worked so I reverted it

File: src/components/ItemDetailPage/ItemDetailPage.tsx [hunk 6]
```diff
@@ -63,7 +63,7 @@ export default class ItemDetailPage extends React.PureComponent<Props> {
       <>
         <Section>
           <Row>
-            <Back onClick={() => onNavigate(locations.collectionDetail(item.collectionId))} />
+            <Back absolute onClick={() => onNavigate(locations.collections())} />
```
[the remark this answers is not in the document]
Comment (nicosantangelo) [reply src/components/ItemDetailPage/ItemDetailPage.tsx:66]: True! it comes from an old version. Updated

[the remark this answers is not in the document]
Comment (nicosantangelo) [reply src/components/ItemDetailPage/ItemDetailPage.tsx:66]: True! it comes from an old version. Updated

File: tsconfig.json [hunk 7]
```diff
@@ -1,8 +1,12 @@
...
   "compilerOptions": {
     "target": "es6",
-    "lib": ["dom", "dom.iterable", "esnext"],
-    "jsx": "preserve",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "jsx": "react-jsx",
```
[the remark this answers is not in the document]
Comment (nicosantangelo) [reply tsconfig.json:9]: That's right, just copied the new tsconfig over. You think we might need to change it?

**Comment by nicosantangelo** [comment]
Updated!

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**